app/src/main/res/layout/activity_students.xml

```diff
@@ -1,5 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v7.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@android:id/list"
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
avoid `RelativeLayout`!! it reduces speed significantly. Replace with `LinearLayout`